apps/src/lib/node/ledger/ethereum_node/events.rs

```diff
@@ -308,7 +311,9 @@ pub mod eth_events {
                         ParamType::Uint(256),
                         ParamType::String,
                         ParamType::Uint(256),
+                        ParamType::String,
```

Comment:
Does this mean that each `transfer` includes the relayer address?
Also, we have to change `TRANSFER_TO_ETHEREUM_SIG`.

Reply:
Good catch. I thought I had removed that.  And I forgot the signature too.